pages/debate.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="/stylesheets/style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About Me</title>
+</head>
+<body>
+  <header></header>
+  <nav>
+    <h1>Emily Saeli</h1>
+    <a href="index.html">Home</a>
+    <a href="about.html">About Me</a>
+    <a href="portfolio.html">Code Portfolio</a>
+    <a href="debate.html">Debate</a>
+    <a href="publications.html">Publications</a>
+    <a href="workexperience.html">Resume</a>
+  </nav>
+
+  <h2>Under construction</h2>
```

Comment:
I think you're legally obligated to put a cheesey construction themed image here.  :-P